### PR TITLE
[TASK] Add issue templates

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

**Review by linawolf — APPROVED** [review]
I think it is a good start, we can still refine it later if we need more or different information.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: 🚀 Feature request
+about: Suggest an idea for this documentation (and may want to implement it 🙂)
+
+---
+
+### Is your feature request related to a problem? Please describe.
+
+A clear and concise description of what the problem is. Ex. I have an issue when [...]
```
Comment (DavidBruchmann) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]:
```suggestion
A clear and concise description about your idea, possibly related to some problem.
```

**Review by DavidBruchmann — CHANGES_REQUESTED** [review]
In feature request the never might be any "problems", so I think the wording should be a bit different.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: 🚀 Feature request
+about: Suggest an idea for this documentation (and may want to implement it 🙂)
+
+---
+
+### Is your feature request related to a problem? Please describe.
```
Comment (brotkrueml) [inline .github/ISSUE_TEMPLATE/feature_request.md:7]:
```suggestion
### Is your feature request related to an issue? Please describe.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+name: 🚀 Feature request
+about: Suggest an idea for this documentation (and may want to implement it 🙂)
+
+---
+
+### Is your feature request related to an issue? Please describe.
+
+A clear and concise description about your idea, possibly related to some problem.
```
Comment (brotkrueml) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]:
```suggestion
A clear and concise description about your idea.
```

**Review by DavidBruchmann — APPROVED** [review]

**Pull request merged**